workshops/spinning_wheel/README.md

```diff
@@ -0,0 +1,404 @@
+---
+name: 'Spin The Wheel'
+description: 'Build a random picker spinning wheel by HTML, CSS and JavaScript'
+author: '@HariOm987'
+img: https://cloud-gvkfrv1qw.vercel.app/0image.png
+---
+You must have seen the spinning wheel by which we can select the random option, like a fortune wheel or a  lucky draw. But have you ever wondered how to make your own wheel by yourself? If you think that it is very tough then I am sorry, you are wrong. Just follow this workshop and you will have your customized spinning wheel in the next 20 minutes. You will surely accomplish this project with the help of this workshop and even if you are stuck at any point then instead of doing this 👇
+
+![person stuck in the problem GIF](https://cloud-b323sylf5.vercel.app/01439996693_lolwutaaa.gif)
+
+Just join [Hack Club Slack](https://hackclub.com/slack/) and ask your doubt in #code channel. There are thousands of high school students who can help you out. 😇
+
+Here's the [live demo](https://movie-selector.hariom04.repl.co/) and [final code](https://repl.it/@hariom04/movie-selector#index.html).
+
+![Final Output of our Workshop](https://cloud-4945tg5q1.vercel.app/0final.png)
+
+^^^ If you will follow this workshop strictly then you will make something like this, but feel free to add up your imaginations with patters and colors and you will end up with something better than this. So let's begin 😁✨
+
+# Part 1 : Prerequisites
+* Account of Repl.it
```

Comment:
grammar

Reply:
Okay, I think now I have corrected that. XD